src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -128,3 +128,35 @@
     </span>
   </td>
 </tr>
+{% if product.customizations is not null %}
+  <tr class="order-product-customization{% if productIndex is defined and paginationNum is defined and productIndex > paginationNum %} d-none{% endif %}">
```

Comment:
```suggestion
  {% set rowIsDiplayed = (productIndex is defined and paginationNum is defined and productIndex > paginationNum) %}
```

May be store the status of the row in a variable for not duplicating the condition.

Reply:
Done